addons/website_slides/static/src/js/slides_delete_slide.js

```diff
@@ -0,0 +1,95 @@
+odoo.define('website_slides.delete.slide', function (require) {
+
+    var sAnimations = require('website.content.snippets.animation');
+    var core = require('web.core');
+    var Widget = require('web.Widget');
+
+    var _t = core._t;
+    var QWeb = core.qweb;
+
+    var DeleteSlideDialog = Widget.extend({
+        template: 'website.slide.delete.slide',
+        events: {
+            'hidden.bs.modal': 'destroy',
+            'click button[data-dismiss="modal"]': '_cancel',
+            'click button.delete': '_delete'
+        },
+
+        /**
+         * @override
+         * @param {Object} el
+         * @param {number} channel_id
+         */
+        init: function (el, slideID) {
+            this._super(el, slideID);
+            this.slide_id = parseInt(slideID, 10);
+        },
+        /**
+         * @override
+         */
+        start: function () {
+            this.$el.modal({
+                backdrop: 'static'
+            });
+            return this._super.apply(this, arguments);
+        },
+        //--------------------------------------------------------------------------
+        // Handlers
+        //--------------------------------------------------------------------------
+        _delete: function (ev) {
+            var self = this;
+            // TO FIX: CallBack is not executed
```

Comment:
Is that still the case?

I think you should only remove the element on success and add a failure handler (something like display the error message somewhere).